fallout/gatling/gatling_timeseries_latency.yaml

```diff
@@ -0,0 +1,99 @@
+# master uses a Gatling DSE plugin based on 3.x
+# DSP-17972-unified-driver uses a Gatling DSE plugin based on the unified driver (>= 4.5.x)
+branch: "master"
+
+---
+
+ensemble:
+  observer:
+    node.count: 1
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: nebula
+        cloud.instance.type: m3.large
+    configuration_manager:
+      - name: ctool_monitoring
+        properties:
+          graphite.create_server: true
+  server:
+    node.count: 3
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: nebula
+        cloud.tenant: moonshot-v2
+        cloud.instance.type: ms2.small
```

Comment:
Nice that we are using the new moonshot-v2 tenant, we'll have to eventually migrate everything there.

Reply:
Hmm I'm unable to access this tenant; how did you create your app credentials?

Reply:
Nevermind, I found it, it's on https://nebula.sjc.dsinternal.org/dashboard/identity/application_credentials/